### PR TITLE
Handle final-size constraint by aspect ratio correctly

### LINKED ISSUE #3268: checkSizeContraint ignored VST3 Mac Reaper
@Kingston1 this is related to your zoom changes I am almost sure

checkSizeConstraint basically lets us force an aspect ratio. I was whacking it to be a no-op on FL to fix #2466 and when testing in reaper noticed that the constraint is no longer forced mac Reaper VST3. I can drag to any ratio and moreover, it never narrows so i can drag to have offscreen elements. I think this is related to your re-order shuffling so wanted to toss this issue in.

I'll push ahead with the disablement for #2466 since that seems like the right fix there no matter what.

### PR DESCRIPTION
**Pull request by baconpaul**
VST3 didn't calculate aspect ratio correctly. We used to
whack that into place with our recursion, bugs and all, but now we don't
this at least stops clipped windows

Closes #3268

### CODE REVIEW

File: src/common/gui/SurgeGUIEditor.cpp [hunk 1]
```diff
@@ -5873,6 +5873,7 @@ void SurgeGUIEditor::resizeFromIdleSentinel()
 {
    if (resizeToOnIdle.x > 10 && resizeToOnIdle.y > 10)
    {
+      std::cout << "Got an idle resize " << resizeToOnIdle.x << " " << resizeToOnIdle.y << std::endl;
```
[the remark this answers is not in the document]
Comment (baconpaul) [reply src/common/gui/SurgeGUIEditor.cpp:5876]: oh shoot i gotta kill that.

[the remark this answers is not in the document]
Comment (baconpaul) [reply src/common/gui/SurgeGUIEditor.cpp]: force pushed a change to remove it
on mac it goes to terminal :)

[the remark this answers is not in the document]
Comment (baconpaul) [reply src/common/gui/SurgeGUIEditor.cpp]: on win if you use printf you can spawn a console from debug menu. evil or i can show you on discord if you want

[the remark this answers is not in the document]
Comment (Kingston1) [reply src/common/gui/SurgeGUIEditor.cpp]: I've used the debug console, just wanted to make sure what happened here.

**Pull request merged**